cmd/juju/cloud/show.go

```diff
@@ -176,7 +160,7 @@ func (c *showCloudCommand) Run(ctxt *cmd.Context) error {
 			}
 		}
 	}
-	if c.BothClientAndController || c.ClientOnly {
+	if c.Client {
```

Comment:
if c.Client && localCloud != nil
Actually, just if localCloud != nil is sufficient right?
And same with remoteCloud != nil above

Reply:
Well, no. I've simplified the section a little.

But basically we need to know whether user has requested a client cloud or not, especially in case where we just did not find it.

This is needed because we are displaying config after the cloud details. If both controller and client clouds were requested and since we display controller clouds first, we need to know if we are showing the config of remote cloud or not.